### PR TITLE
Team-3 error fixed

### PR DESCRIPTION
**Pull request by Moksh-Mehta7**
Game running now

### CODE REVIEW

**Review by Mohamad11Dab — COMMENTED** [review]
Can you guys please do another merge from main and check that it does not crash because somebody pushed their commits to main just after your pr

File: source/core/src/main/com/csse3200/game/areas/ForestGameArea.java [hunk 1]
```diff
@@ -213,7 +213,6 @@ public void create() {
   private void displayUI() {
     Entity ui = new Entity();
     ui.addComponent(new GameAreaDisplay("Box Forest"));
-    ui.addComponent(ServiceLocator.getGameEndService().getDisplay());
```
Comment (The-AhmadAA) [inline source/core/src/main/com/csse3200/game/areas/ForestGameArea.java:216]: You have deleted a change required by another team's feature. Restore this and fix your error properly

**Review by The-AhmadAA — CHANGES_REQUESTED** [review]
GameEndService is someone elses feature and you should not be modifying it. See #132

**Comment by Hasakev** [comment]
image
Hi this is not intended, please fix

**Comment by Gaganx0** [comment]
We are removing this feature from this sprint due to time constraints.

**Pull request closed without merging**